spark/v3.5/spark-runtime/LICENSE

```diff
@@ -403,22 +350,6 @@ License: http://www.apache.org/licenses/LICENSE-2.0
 
 --------------------------------------------------------------------------------
 
-This binary artifact contains findbugs-annotations by Stephen Connolly.
```

Comment:
I don't find findbugs/jsr305 in the spark-runtime jar.

Reply:
It looks like this is mostly unused now. We exclude jsr305 and don't replace it with this anymore.

Reply:
Yup, that's why I removed the mention from `LICENSE`.